tutorials/convert_lit_models.md

```diff
@@ -115,5 +115,36 @@ import torch
 from transformers import AutoModel
 
 state_dict = torch.load('out/hf-tinyllama/converted/model.pth')
-model = AutoModel.from_pretrained("TinyLlama/TinyLlama-1.1B-intermediate-step-1431k-3T", state_dict=state_dict)
+model = AutoModel.from_pretrained("checkpoints/TinyLlama/TinyLlama-1.1B-intermediate-step-1431k-3T", state_dict=state_dict)
```

Comment:
Is this change right? Isn't this meant to be the repo id?

Reply:
Yes, thanks

Reply:
I think you forgot to push the update

```suggestion
model = AutoModel.from_pretrained("TinyLlama/TinyLlama-1.1B-intermediate-step-1431k-3T", state_dict=state_dict)
```

Reply:
Hm weird, I remember pressing the "commit suggestion" button. Maybe I accidentally overwrote it with my local version afterwards. Should be fixed now though